index.html

```diff
@@ -1,120 +1,77 @@
 <!doctype html>
 <html>
-	
+
 	<head>
-		<link rel="stylesheet" href="stylesheet.css"/>
-		<link rel="shortcut icon" href="favicon.ico" type="image/x-icon"/>
+		<link rel="stylesheet" href="https://maxcdn.bootstrapcdn.com/bootstrap/4.0.0/css/bootstrap.min.css" integrity="sha384-Gn5384xqQ1aoWXA+058RXPxPg6fy4IWvTNh0E263XmFcJlSAwiGgFAW/dAiS6JXm" crossorigin="anonymous">
+		<link rel="shortcut icon" href="./images/favicon.ico" type="image/x-icon"/>
 		<title>Smelt: Batch Package Manager</title>
+		<style>
+		code {
+			color:#3498db;
+		}
+		</style>
 	</head>
-	
-	
+
+
 <body>
-		
-	<div class="navbar">
-	<br/>
-        <h1>Smelt: Batch Package Manager</h1>
-	<br/>
-        </div>
-	
-	<br/>
-	<br/>
-	<br/>
-	
-	<div class="boxes">
-	<img src="windows.png">
-	<h2>Windows Installation</h2>
-	<h3>Install this package manager</h3>
-	
-		
-	<ol>
-	<li>Download <a href="https://smelt-modding.github.io/smelt/setup.bat"><code>setup.bat</code></a>.</li>
-	<li>Double-click it.</li>
-	<li>You're done! There will be:<ul>
-	<li>two new folders: 
-	<br/><code>extension</code> and <code>logs</code></li>
-	<li>and five new files:</l1>
-	<br/>
-	<code>sbpm.bat, install.bat, uninstall.bat, update.cmd, and reinstall.bat</code>
-	</ul>
-	</li>
-	</ol>
-	</div>
 
-	
-	<div class="boxes">
-	<h3>To install commands pre-downloaded from other sources:</h3>
-	<ol>
-	<li>Move the downloaded commands into the <code>extensions</code> folder.</li>
-	<li>Double-click <code>install.bat</code></li>
-	<li>You're done! A log will be available at <code>logs\install.log</code>.</li>
-	</ol>
-	</div>
-		
-	<div class="boxes">
-	<h3>To install commands available here:</h3>
-	<ol>
-	<li>Open Command Prompt (Win+R, type <code>cmd</code> and hit Enter/Win+X, C)</li>
-	<li>Type <code>sbpm install &lt;package></code> (replace <code>&lt;package></code> with the name of the desired package.)</li>
-	<li>You're done! A log will be available at <code>logs\install.log</code>.</li>
-	</ol>
-	</div>
-				
-	
-	<div class="boxes">
-        <h3>To upgrade commands pre-downloaded from other sources:</h3>
-	<ol>
-	<li>Delete the previous versions from the <code>extension</code> folder.</li>
-	<li>Move the new versions into the <code>extension</code> folder.</li>
-	<li>Double-click <code>reinstall.bat</code></li>
-	<li>You're done! Logs will be available at <code>logs\uninstall.log</code> and <code>logs\install.log</code>.</li>
-	</ol>
+	<!-- Smelt Navbar -->
```

Comment:
HTML comments, nice

Reply:
lol, It's quite a long HTML file with a lot of ``<div>``'s, So it's better to know exactly where everything starts and ends.